src/wdio/browser.ts

```diff
@@ -85,4 +88,17 @@ export class Browser {
     public static async waitUntil(condition: () => boolean | Promise<boolean>, timeout: number = DefaultTimeout.SLOW_WAIT): Promise<void> {
         await browser.waitUntil(condition, {timeout});
     }
+
+
+    // Screenshots
+    public static async takeScreenshot(): Promise<string> {
```

Comment:
Question: Just curiosity. Why do we have takeScreenshot and saveScreenshot methods duplicated on the element-finder.ts and browser?

Reply:
Just to have the possibility to takeScreenshot and/or saveScreenshot from the whole Browser object or just from a specific ElementFinder